xarray/core/options.py

```diff
@@ -290,9 +290,16 @@ def __exit__(self, type, value, traceback):
         self._apply_update(self.old)
 
 
-def get_options():
+def get_options(*, join: str = 'inner', copy: bool = True,
+                indexes: Tuple[str, ...] = None,
+                exclude: FrozenSet[str] = frozenset(),
+                fill_value: Any = None) -> FrozenDict[str, Any]):
```

Comment:
```suggestion
                fill_value: Any = None) -> FrozenDict[str, Any]:
```
But actually I have no idea what is the purpose of these arguments? They are not used anywhere!